### PR TITLE
[12.0][MIG] account_invoice_margin_sale: Migration to 12.0

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa
TT20159

### CODE REVIEW

**Comment by carlosdauden** [comment]
Depends https://github.com/OCA/margin-analysis/pull/66

**Comment by pedrobaeza** [comment]
Please rebase now that the dependency is merged.

**Review by sergio-teruel — COMMENTED** [review]
Waiting for green ci

File: account_invoice_margin_sale/__manifest__.py
Comment (pedrobaeza) [inline account_invoice_margin_sale/__manifest__.py]: When promoting to stable, you have to add at least one maintainer

**Review by pedrobaeza — APPROVED** [review]

File: account_invoice_margin_sale/tests/test_account_invoice_margin_sale.py
Comment (sergio-teruel) [inline account_invoice_margin_sale/tests/test_account_invoice_margin_sale.py]: Please use a new pricelist to avoid conflicts with other module tests

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Comment by carlosdauden** [comment]
Changes done

**Comment by pedrobaeza** [comment]
@sergio-teruel do you approve it?

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**
